Godeps/Godeps.json

```diff
@@ -0,0 +1,15 @@
+{
+	"ImportPath": "github.com/Clever/gearman-load-logger",
+	"GoVersion": "go1.3",
+	"Deps": [
+		{
+			"ImportPath": "github.com/Clever/gearadmin",
+			"Rev": "24bb3bffb82cc94a5c2df192d483667130d55942"
+		},
+		{
+			"ImportPath": "gopkg.in/Clever/kayvee-go.v0",
+			"Comment": "v0.0.1",
```

Comment:
may as well bump this to `v0.0.3` as that is what will actually be imported (unless you want to goto v1 ? )